cdmtaskservice/errors.py

```diff
@@ -0,0 +1,44 @@
+"""
+Error codes for exceptions thrown by the CDM task service system.
```

Comment:
most of the error codes will be used in the future?

Reply:
Yep, I'd expect so. If not I'll delete them